R/zchunk_L101.nonghg_en_USA_S_T_Y.R

```diff
@@ -59,18 +59,39 @@ module_emissions_L101.nonghg_en_USA_S_T_Y <- function(command, ...) {
     EPA_VOC <- get_data(all_data, "emissions/EPA_VOC")
     EPA_NH3 <- get_data(all_data, "emissions/EPA_NH3")
 
+    # get_data calls followed by temporary (temp-data-inject) reshapes
+    L1322.in_EJ_R_indenergy_F_Yh <- get_data(all_data, "temp-data-inject/L1322.in_EJ_R_indenergy_F_Yh") %>%
+      gather(year, value, -GCAM_region_ID, -sector, -fuel) %>%
+      mutate(year = as.integer(substr(year, 2, 5)))
+
     L1231.in_EJ_R_elec_F_tech_Yh <- get_data(all_data, "L1231.in_EJ_R_elec_F_tech_Yh")
-    L1322.in_EJ_R_indenergy_F_Yh <- get_data(all_data, "temp-data-inject/L1322.in_EJ_R_indenergy_F_Yh")
+
+    # The following two data frames can't be reshaped from long to wide by the test system
+    # So leave them wide, and strip off X's
```

Comment:
Weird.  What does having multiple entries even mean?  Is there some distinguishing characteristic between them that isn't captured in the table?